flink-python/pyflink/table/table_environment.py

```diff
@@ -98,18 +99,25 @@ def __init__(self, j_tenv, serializer=PickleSerializer()):
         self._open()
 
     @staticmethod
-    def create(environment_settings: EnvironmentSettings) -> 'TableEnvironment':
+    def create(conf_or_settings: Union[EnvironmentSettings, Configuration]) -> 'TableEnvironment':
```

Comment:
This will break backward compatibility. Users may write code as following `TableEnvironment.create(environment_settings=env_settings)`. What about keep the name not changed?